<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qpwgraph - A PipeWire Graph Qt GUI Interface

   Copyright (C) 2021-2025, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qpwgraph_options</class>
 <widget class="QDialog" name="qpwgraph_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qpwgraph.qrc">:/images/qpwgraph.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>240</height>
   </size>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="OptionsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="GeneralTabPage">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="SystemTrayEnabledCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to enable the system tray icon</string>
         </property>
         <property name="text">
          <string>&amp;Enable system tray icon</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" rowspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="SystemTrayQueryCloseCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to show system tray message on main window close</string>
         </property>
         <property name="text">
          <string>Show &amp;system tray message on close</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="SystemTrayStartMinimizedCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to start minimized to system tray</string>
         </property>
         <property name="text">
          <string>Start &amp;minimized to system tray</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="PatchbayQueryQuitCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to show activated patchbay message on application quit</string>
         </property>
         <property name="text">
          <string>Show activated &amp;patchbay message on quit</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QCheckBox" name="AlsaMidiEnabledCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to enable ALSA MIDI support</string>
         </property>
         <property name="text">
          <string>Enable &amp;ALSA MIDI support</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FilterNodesTabPage">
      <attribute name="title">
       <string>Filter</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="FilterNodesEnabledCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to filter / hide nodes from the graph</string>
         </property>
         <property name="text">
          <string>Enable filter / hide nodes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="FilterNodesNameComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Node name (regex)</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="FilterNodesAddToolButton">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Add node name (regex) to filter / hide list</string>
         </property>
         <property name="text">
          <string>&amp;Add</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="3">
        <widget class="QListWidget" name="FilterNodesListWidget">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Node name (regex) filter / hide list</string>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="FilterNodesRemoveToolButton">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Remove node name (regex) from filter / hide list</string>
         </property>
         <property name="text">
          <string>&amp;Remove</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="FilterNodesClearToolButton">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Clear filter / hide list</string>
         </property>
         <property name="text">
          <string>&amp;Clear</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MergerNodesTabPage">
      <attribute name="title">
       <string>Merger</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="MergerNodesEnabledCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to merge / unify nodes on the patchbay</string>
         </property>
         <property name="text">
          <string>Enable merge / unify nodes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="MergerNodesNameComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Node name (regex)</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="MergerNodesAddToolButton">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Add node name (regex) to merger / unify list</string>
         </property>
         <property name="text">
          <string>&amp;Add</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="3">
        <widget class="QListWidget" name="MergerNodesListWidget">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Node name (regex) merger / unify list</string>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="MergerNodesRemoveToolButton">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Remove node name (regex) from merger / unify list</string>
         </property>
         <property name="text">
          <string>&amp;Remove</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="MergerNodesClearToolButton">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Clear merger / unify list</string>
         </property>
         <property name="text">
          <string>&amp;Clear</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8"/>
 <tabstops>
  <tabstop>OptionsTabWidget</tabstop>
  <tabstop>SystemTrayEnabledCheckBox</tabstop>
  <tabstop>SystemTrayQueryCloseCheckBox</tabstop>
  <tabstop>SystemTrayStartMinimizedCheckBox</tabstop>
  <tabstop>PatchbayQueryQuitCheckBox</tabstop>
  <tabstop>AlsaMidiEnabledCheckBox</tabstop>
  <tabstop>FilterNodesEnabledCheckBox</tabstop>
  <tabstop>FilterNodesNameComboBox</tabstop>
  <tabstop>FilterNodesAddToolButton</tabstop>
  <tabstop>FilterNodesListWidget</tabstop>
  <tabstop>FilterNodesRemoveToolButton</tabstop>
  <tabstop>FilterNodesClearToolButton</tabstop>
  <tabstop>MergerNodesEnabledCheckBox</tabstop>
  <tabstop>MergerNodesNameComboBox</tabstop>
  <tabstop>MergerNodesAddToolButton</tabstop>
  <tabstop>MergerNodesListWidget</tabstop>
  <tabstop>MergerNodesRemoveToolButton</tabstop>
  <tabstop>MergerNodesClearToolButton</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qpwgraph.qrc"/>
 </resources>
 <connections/>
</ui>
